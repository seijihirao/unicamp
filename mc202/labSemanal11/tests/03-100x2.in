200
i 112 112
i 195 195
i 137 137
i 142 142
i 116 116
i 130 130
i 152 152
i 184 184
i 158 158
i 186 186
i 121 121
i 199 199
i 111 111
i 123 123
i 162 162
i 178 178
i 135 135
i 192 192
i 172 172
i 113 113
i 133 133
i 153 153
i 12  12
i 194 194
i 145 145
i 114 114
i 139 139
i 187 187
i 157 157
i 180 180
i 198 198
i 150 150
i 134 134
i 110 110
i 140 140
i 154 154
i 141 141
i 196 196
i 19  19
i 176 176
i 147 147
i 115 115
i 143 143
i 175 175
i 117 117
i 183 183
i 179 179
i 185 185
i 182 182
i 138 138
i 120 120
i 164 164
i 161 161
i 149 149
i 131 131
i 165 165
i 193 193
i 13  13
i 181 181
i 127 127
i 122 122
i 10  10
i 14  14
i 190 190
i 155 155
i 144 144
i 136 136
i 146 146
i 166 166
i 168 168
i 177 177
i 118 118
i 167 167
i 129 129
i 16  16
i 132 132
i 169 169
i 15  15
i 18  18
i 174 174
i 156 156
i 125 125
i 160 160
i 126 126
i 170 170
i 163 163
i 189 189
i 171 171
i 151 151
i 159 159
i 197 197
i 11  11
i 124 124
i 17  17
i 173 173
i 148 148
i 119 119
i 191 191
i 188 188
i 128 128
d 112 12
d 195 95
d 137 37
d 142 42
d 116 16
d 130 30
d 152 52
d 184 84
d 158 58
d 186 86
d 121 21
d 199 99
d 111 11
d 123 23
d 162 62
d 178 78
d 135 35
d 192 92
d 172 72
d 113 13
d 133 33
d 153 53
d 12  2
d 194 94
d 145 45
d 114 14
d 139 39
d 187 87
d 157 57
d 180 80
d 198 98
d 150 50
d 134 34
d 110 10
d 140 40
d 154 54
d 141 41
d 196 96
d 19  9
d 176 76
d 147 47
d 115 15
d 143 43
d 175 75
d 117 17
d 183 83
d 179 79
d 185 85
d 182 82
d 138 38
d 120 20
d 164 64
d 161 61
d 149 49
d 131 31
d 165 65
d 193 93
d 13  3
d 181 81
d 127 27
d 122 22
d 10  0
d 14  4
d 190 90
d 155 55
d 144 44
d 136 36
d 146 46
d 166 66
d 168 68
d 177 77
d 118 18
d 167 67
d 129 29
d 16  6
d 132 32
d 169 69
d 15  5
d 18  8
d 174 74
d 156 56
d 125 25
d 160 60
d 126 26
d 170 70
d 163 63
d 189 89
d 171 71
d 151 51
d 159 59
d 197 97
d 11  1
d 124 24
d 17  7
d 173 73
d 148 48
d 119 19
d 191 91
d 188 88
d 128 28
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
i 112 -112
i 195 -195
i 137 -137
i 142 -142
i 116 -116
i 130 -130
i 152 -152
i 184 -184
i 158 -158
i 186 -186
i 121 -121
i 199 -199
i 111 -111
i 123 -123
i 162 -162
i 178 -178
i 135 -135
i 192 -192
i 172 -172
i 113 -113
i 133 -133
i 153 -153
i 12  -12
i 194 -194
i 145 -145
i 114 -114
i 139 -139
i 187 -187
i 157 -157
i 180 -180
i 198 -198
i 150 -150
i 134 -134
i 110 -110
i 140 -140
i 154 -154
i 141 -141
i 196 -196
i 19  -19
i 176 -176
i 147 -147
i 115 -115
i 143 -143
i 175 -175
i 117 -117
i 183 -183
i 179 -179
i 185 -185
i 182 -182
i 138 -138
i 120 -120
i 164 -164
i 161 -161
i 149 -149
i 131 -131
i 165 -165
i 193 -193
i 13  -13
i 181 -181
i 127 -127
i 122 -122
i 10  -10
i 14  -14
i 190 -190
i 155 -155
i 144 -144
i 136 -136
i 146 -146
i 166 -166
i 168 -168
i 177 -177
i 118 -118
i 167 -167
i 129 -129
i 16  -16
i 132 -132
i 169 -169
i 15  -15
i 18  -18
i 174 -174
i 156 -156
i 125 -125
i 160 -160
i 126 -126
i 170 -170
i 163 -163
i 189 -189
i 171 -171
i 151 -151
i 159 -159
i 197 -197
i 11  -11
i 124 -124
i 17  -17
i 173 -173
i 148 -148
i 119 -119
i 191 -191
i 188 -188
i 128 -128
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
m
t
