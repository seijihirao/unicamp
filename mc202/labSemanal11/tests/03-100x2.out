minimo {10,0}
minimo {11,1}
minimo {12,2}
minimo {13,3}
minimo {14,4}
minimo {15,5}
minimo {16,6}
minimo {17,7}
minimo {18,8}
minimo {19,9}
minimo {110,10}
minimo {111,11}
minimo {112,12}
minimo {113,13}
minimo {114,14}
minimo {115,15}
minimo {116,16}
minimo {117,17}
minimo {118,18}
minimo {119,19}
minimo {120,20}
minimo {121,21}
minimo {122,22}
minimo {123,23}
minimo {124,24}
minimo {125,25}
minimo {126,26}
minimo {127,27}
minimo {128,28}
minimo {129,29}
minimo {130,30}
minimo {131,31}
minimo {132,32}
minimo {133,33}
minimo {134,34}
minimo {135,35}
minimo {136,36}
minimo {137,37}
minimo {138,38}
minimo {139,39}
minimo {140,40}
minimo {141,41}
minimo {142,42}
minimo {143,43}
minimo {144,44}
minimo {145,45}
minimo {146,46}
minimo {147,47}
minimo {148,48}
minimo {149,49}
minimo {150,50}
minimo {151,51}
minimo {152,52}
minimo {153,53}
minimo {154,54}
minimo {155,55}
minimo {156,56}
minimo {157,57}
minimo {158,58}
minimo {159,59}
minimo {160,60}
minimo {161,61}
minimo {162,62}
minimo {163,63}
minimo {164,64}
minimo {165,65}
minimo {166,66}
minimo {167,67}
minimo {168,68}
minimo {169,69}
minimo {170,70}
minimo {171,71}
minimo {172,72}
minimo {173,73}
minimo {174,74}
minimo {175,75}
minimo {176,76}
minimo {177,77}
minimo {178,78}
minimo {179,79}
minimo {180,80}
minimo {181,81}
minimo {182,82}
minimo {183,83}
minimo {184,84}
minimo {185,85}
minimo {186,86}
minimo {187,87}
minimo {188,88}
minimo {189,89}
minimo {190,90}
minimo {191,91}
minimo {192,92}
minimo {193,93}
minimo {194,94}
minimo {195,95}
minimo {196,96}
minimo {197,97}
minimo {198,98}
minimo {199,99}
vazio
minimo {199,-199}
minimo {198,-198}
minimo {197,-197}
minimo {196,-196}
minimo {195,-195}
minimo {194,-194}
minimo {193,-193}
minimo {192,-192}
minimo {191,-191}
minimo {190,-190}
minimo {189,-189}
minimo {188,-188}
minimo {187,-187}
minimo {186,-186}
minimo {185,-185}
minimo {184,-184}
minimo {183,-183}
minimo {182,-182}
minimo {181,-181}
minimo {180,-180}
minimo {179,-179}
minimo {178,-178}
minimo {177,-177}
minimo {176,-176}
minimo {175,-175}
minimo {174,-174}
minimo {173,-173}
minimo {172,-172}
minimo {171,-171}
minimo {170,-170}
minimo {169,-169}
minimo {168,-168}
minimo {167,-167}
minimo {166,-166}
minimo {165,-165}
minimo {164,-164}
minimo {163,-163}
minimo {162,-162}
minimo {161,-161}
minimo {160,-160}
minimo {159,-159}
minimo {158,-158}
minimo {157,-157}
minimo {156,-156}
minimo {155,-155}
minimo {154,-154}
minimo {153,-153}
minimo {152,-152}
minimo {151,-151}
minimo {150,-150}
minimo {149,-149}
minimo {148,-148}
minimo {147,-147}
minimo {146,-146}
minimo {145,-145}
minimo {144,-144}
minimo {143,-143}
minimo {142,-142}
minimo {141,-141}
minimo {140,-140}
minimo {139,-139}
minimo {138,-138}
minimo {137,-137}
minimo {136,-136}
minimo {135,-135}
minimo {134,-134}
minimo {133,-133}
minimo {132,-132}
minimo {131,-131}
minimo {130,-130}
minimo {129,-129}
minimo {128,-128}
minimo {127,-127}
minimo {126,-126}
minimo {125,-125}
minimo {124,-124}
minimo {123,-123}
minimo {122,-122}
minimo {121,-121}
minimo {120,-120}
minimo {119,-119}
minimo {118,-118}
minimo {117,-117}
minimo {116,-116}
minimo {115,-115}
minimo {114,-114}
minimo {113,-113}
minimo {112,-112}
minimo {111,-111}
minimo {110,-110}
minimo {19,-19}
minimo {18,-18}
minimo {17,-17}
minimo {16,-16}
minimo {15,-15}
minimo {14,-14}
minimo {13,-13}
minimo {12,-12}
minimo {11,-11}
minimo {10,-10}
vazio
